index.html

```diff
@@ -42,5 +46,271 @@ <h1 class=" bigger-text">I'm Albert. Glad to see you!</h1>
+              </div>
+              <div>
+                  <p class="about-text">
+                      Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                      If you like what you see and have a project you need coded, don't hestiate to contact me.
+                  </p>
+              </div>
+              <div>
+                  <h3 class="connect">
+                      LET'S CONNECT
+                  </h3>
+                  <div>
+                      <ul class="social">
+                          <li>
+                              <a href="#"><img src="Vector@2x.png" alt="Vector" width="17px" height="20px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="Linkedin icon@2x.png" alt="Linkedin" width="20px" height="20px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="vecto@2x.png" alt="vecto" width="14px" height="20px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="twitter.png" alt="twitter" width="19.99px" height="19.99px"></a>
+                          </li>
+                          <li>
+                              <a href="#"><img src="media.png" alt="media" width="20px" height="16px"></a>
+                          </li>
+                      </ul>
+                  </div>
+              </div>
+              <div>
+                <button class="see-project-button" type="button">Get my resume</button>
+              </div>
+              <ul class="about-list">
+                <li class="tab-item">
+                    <div class="language-title"><span class="tab-text language-tittle">Languages <i class='fas fast'>&#xf078;</i></span></div>
+                    <ul class="tab-langs">
+                        <li class="frame-lang frame">
+                          <img src="js.png"><span class="tab-text">JavaScript</span>
+                        </li>
+                        <li class="frame-lang">
+                          <img src="html.png"><span class="tab-text">HTML</span>
+                        </li>
+                        <li class="frame-lang">
+                          <img src="css.png"><span class="tab-text">CSS</span>
+                        </li>
+                    </ul>
+                </li>
+                <li class="tab-item">
+                    <span class="language-title">Frameworks<i class='fas'>&#xf054;</i></span>
+                    <div class="line"></div>
+                </li>
+                <li class="tab-item">
+                    <span class="language-title skill">Skills<i class='fas'>&#xf054;</i></span>
+                    <div class="line"></div>
+                </li>
+            </ul> 
+          </div>
+      </div>
+  </section>
```

Comment:
- [ ] The previous reviewer requested that the About section be properly laid out. Please work on this to ensure that your design is more similar to the original design. :ok_hand: 😊
### You design
![Screenshot from 2023-05-06 18-31-24](https://user-images.githubusercontent.com/89228743/236641060-c84d8bb3-22c9-4c5f-8334-d0485fffbea2.png)

To assist you, I will make suggestions for all of the elements whose styles you should modify. 

### Suggestions: These are styles I just adjust few things

1. 

```
.connect {
  font-style: normal;
  font-weight: 500;
  font-size: 16px;
  line-height: 24px;
  color: #7f8cff;
  margin: 0;
}
```
2.
```
.social {
  display: flex;
  justify-content: flex-start;
  column-gap: 20px;
  padding: 12px 0 0 10px;
  list-style: none;
}
```
3.
```
.see-project-button {
  color: #6070ff;
  background: transparent;
  padding: 12px;
  border: 1px solid #6070ff;
  border-radius: 8px;
  font-size: 17px;
  font-weight: 500;
  line-height: 24px;
  cursor: pointer;
}
```
4.
```
.tab-langs {
  padding: 0;
}
```

After applying the suggestion your desing will look like this :point_down:  which is more like the Figma design. :ok_hand: 
![Screenshot from 2023-05-06 18-22-37](https://user-images.githubusercontent.com/89228743/236641146-bda9bc9d-b34a-45ce-b5c9-15586c6671bd.png)

### I hope this is going to help 😊

Reply:
To make it much easier for you, [this](https://codeshare.io/78ojJk) is your style with all the suggestions. You can copy it and replace it with what you have in your style file.